hooks/post_plan.py

```diff
@@ -102,31 +127,131 @@ def _validate_security_groups(
                     f"Security group {sg.get('GroupId')} does not belong to the same VPC as the subnets"
                 )
 
-    def _validate_parameter_group(self, name: str) -> None:
+    def _validate_engine_version(self, engine: str, engine_version: str) -> str | None:
```

Comment:
I'm having second thoughts on naming conventions here. a "validate" function should return a bool right? But since is not a public method I think we can let this pass. Not a blocker.

Reply:
You know, naming things is the most complicated thing :) It's returning the engine family, so that the `_validate_family...` method doesn't need to fetch the information again.